events/c_3_project_and_community.toml

```diff
@@ -81,9 +81,9 @@ description="A walkthrough of the new RFC process and its motivations, updated s
 
 [[timeslots]]
 startTime="09:45"
-speakers=[]
-title="Open Slot"
-description=""
+speakers=["@galargh"]
+title="GitHub Management"
+description="A high-level overview of how GitHub permissions are managed as code in the IPFS org - https://github.com/ipfs/github-mgmt - and how to leverage it for issuing access requests."
```

Comment:
I rephrased the description to be more clear too and included the information that the focus will be on IPFS org.

As for the context, I added an example situation when it might come in handy - `if you contribute frequently to a repository within IPFS and you'd want to become a collaborator`.

Let me know what you think :)